oiosaml/src/main/java/dk/gov/oio/saml/filter/AuthenticatedFilter.java

```diff
@@ -111,6 +113,13 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
 
                 req.getSession().setAttribute(Constants.SESSION_REQUESTED_PATH, reqPath);
                 MessageContext<SAMLObject> authnRequest = authnRequestService.createMessageWithAuthnRequest(isPassive, forceAuthn, requiredNsisLevel, attributeProfile);
+
+                OIOSAML3Service.getAuditService().auditLog(new AuditService
```

Comment:
At least missing remote port and HTTP session identifier in object.

Reply:
Also it must be configurable where the remote IP and port are found in the request.

Reply:
I have "completed" the BSA flow, take a look and review